packages/yourloops/test/integration/pages/home-page/caregiver-home-page.spec.tsx

```diff
@@ -142,14 +142,13 @@ describe('Caregiver home page', () => {
     })
 
     await checkCaregiverLayout(`${firstName} ${lastName}`)
-    checkSecondaryBar(false, false)
+    checkPatientList(UserRole.Caregiver)
 
-    const patientTableBody = within(screen.getByTestId('patient-table-body'))
+    const patientTableBody = within(screen.getByTestId('patient-list'))
     const patientData = patientTableBody.getByText(patientFullName)
     expect(patientData).toBeVisible()
 
-    const patientRow = screen.queryByTestId(`patient-row-${unmonitoredPatientAsTeamMember.userId}`)
-    const removePatientButton = within(patientRow).getByRole('button', { name: 'Remove patient unmonitored-patient@diabeloop.fr' })
+    const removePatientButton = screen.getByRole('button', { name: `Remove patient ${unmonitoredPatientAsTeamMember.userId}` })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏userId ? It would be way better to have the user email instead (as before)